fdbclient/include/fdbclient/ServerKnobs.h

```diff
@@ -188,12 +188,19 @@ class ServerKnobs : public KnobsImpl<ServerKnobs> {
 	int PRIORITY_ENFORCE_MOVE_OUT_OF_PHYSICAL_SHARD;
 
 	// Data distribution
-	// DD won't move shard to teams that has availableSpaceRatio < max(0.05,  AllTeamAvailSpaceRatio[pivot]), where
-	// pivot = pivot percent * team count.
-	double AVAILABLE_SPACE_PIVOT_PERCENT;
-	// DD won't move shard to teams that has readLoad > AllTeamReadLoad[pivot], where pivot = pivot percent *
+	// DD use AVAILABLE_SPACE_PIVOT_RATIO to calculate pivotAvailableSpaceRatio. Given a array that's descend
+	// sorted by available space ratio, the pivot position is AVAILABLE_SPACE_PIVOT_RATIO * team count.
+	// When pivotAvailableSpaceRatio is lower than TARGET_AVAILABLE_SPACE_RATIO, the DD won't move any shard to the team
+	// has available space ratio < pivotAvailableSpaceRatio.
+	double AVAILABLE_SPACE_PIVOT_RATIO;
+	// DD won't move shard to teams that has CPU > AllTeamCPUAscend[pivot], where pivot = CPU_PIVOT_RATIO *
 	// team count.
-	double READ_LOAD_PIVOT_PERCENT;
+	double CPU_PIVOT_RATIO;
+	// DD won't move shard to teams that has CPU > MAX_DEST_CPU_PERCENT
+	double MAX_DEST_CPU_PERCENT;
+	// The constant interval DD update pivot values for team selection. It should be >=
+	// min(STORAGE_METRICS_POLLING_DELAY,DETAILED_METRIC_UPDATE_RATE)  otherwise the pivot won't change;
+	double DD_TEAM_PIVOT_UPDATE_DELAY;
```

Comment:
I've thought about the same. I gave up the effort because this section is under `//Data Distribution` section and `MIN_AVAILABLE_* and TARGET_AVAILABLE_SPACE_RATIO` are under `//Ratekeeper` section (they are used by DD and Ratekeeper). Probably the current status is the best we can do?